threshold_tools_application_examples.ipynb

```diff
@@ -49,26 +49,29 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "627b7fad-86f3-484b-9ff5-d11631fe575c",
+   "id": "470105c8-f8d9-4291-843c-9a450da097e9",
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
-    "!pip install git+https://github.com/OpenHydrology/lmoments3.git"
+    "import climakitae as ck\n",
+    "from climakitae import threshold_tools"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "470105c8-f8d9-4291-843c-9a450da097e9",
-   "metadata": {
-    "tags": []
-   },
+   "id": "d002c171-4832-4faa-a6e7-1219431b4f65",
+   "metadata": {},
    "outputs": [],
    "source": [
-    "import climakitae as ck\n",
-    "from climakitae import threshold_tools"
+    "from dask.distributed import progress\n",
+    "from dask_gateway import GatewayCluster\n",
+    "cluster = GatewayCluster()\n",
+    "cluster.adapt(minimum=0, maximum=16)\n",
+    "client = cluster.get_client()\n",
+    "cluster"
```

Comment:
Might be nice to have a line of text above the cluster code to explain what's happening, i.e. "Next, spin up a compute cluster to speed up cell operations"

Reply:
^ now that the new web content is up I was planning to go back and put links to our technical FAQ with dask cluster chunks too:

'Next, spin up a compute cluster to speed up cell operations. To learn more about dask clusters and troubleshoot common issues [see our technical FAQ](https://analytics.cal-adapt.org/docs/faq/).'

Reply:
@gdicecco - does this need to go in all of our notebooks that use clusters? I would say yes....

Reply:
Yes, that's the plan. Also a sentence in the intro sections of each one linking to the data catalog. Maybe we can discuss at standup parking lot the best way to structure those PRs.

Reply:
Agreed. Just for consistency purposes, that might be a separate PR(s)

Reply:
Intentionally not doing this markdown update at this time, as we will be doing an all-notebook-update on this markdown language as our next step.